arch/sim/src/sim/posix/sim_rawgadget.c

```diff
@@ -621,8 +638,13 @@ static void *host_raw_ephandle(void *arg)
   struct usb_raw_gadget_dev_t *dev = &g_raw_gadget_dev;
   struct usb_raw_ep_entry_s *entry = arg;
   struct usb_raw_data_io_s *io;
+  struct sigaction action;
 
-  while (dev->fd >= 0)
+  memset(&action, 0, sizeof(struct sigaction));
```

Comment:
```suggestion
  memset(&action, 0, sizeof(action));
```

(To future-proof the code against type of `action` changing...)

Reply:
Done~